optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/statistic/PureSubSingleStatistic.java

```diff
@@ -39,7 +39,8 @@
         PickedMoveTypeBestScoreDiffSubSingleStatistic.class,
         PickedMoveTypeStepScoreDiffSubSingleStatistic.class
 })
-public abstract class PureSubSingleStatistic<P extends StatisticPoint> extends SubSingleStatistic<P> {
+public abstract class PureSubSingleStatistic<Solution_, P extends StatisticPoint>
```

Comment:
Can we rename P to StatisticPoint_ ?
The idea is to get generic types that are self-explanatory by using the superclassname + "_"